python/ray/serve/backend_state.py

```diff
@@ -837,8 +864,14 @@ def _scale_backend_replicas(
         """
         assert (backend_tag in self._backend_metadata
                 ), "Backend {} is not registered.".format(backend_tag)
-        assert target_replicas >= 0, ("Number of replicas must be"
-                                      " greater than or equal to 0.")
+        if isinstance(target_replicas, str):
+            assert target_replicas == "EveryNode"
```

Comment:
sorry i don't have full context on this concept yet .. does "EveryNode" means the target replica num == all nodes we have in the ray cluster ? If so it might be simpler to handle it outside of this function call and simply pass in a number.